index.html

```diff
@@ -1,17 +1,20 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
     <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link href="style.css" rel="stylesheet" />
+        <title>Microverse Second Trial</title>
     </head>
     <body>
-        <div id="main">
+        <main>
             <h1 id="title">Dr. Norman Borlaug</h1>
             <h4 id="sub-title">The man who saved a billion lives</h4>
-            <div id="img-div">
-                <img src="image.jpg" width="100%" height="auto" style="align-content: center;"><br><br>
+            <section id="img-div">
+                <img src="image.jpg" alt="main-image"><br><br>
```

Comment:
Use margin in CSS instead of br tags in HTML

Reply:
Nicely done :+1: